### ISSUE TITLE
delay sending email?

### ISSUE THREAD

**Issue by robingantois** [anoved/OctoPrint-EmailNotifier#49]
Hi all, I am looking for a solution to the next problem:
I have a setup for my Prusa MK2s and Octopi sending an e-mail to IFTTT to trigger an off-outlet switch via Telldus live. Now I want the OctoPrint-EmailNotifier plugin to send its mail after 300 seconds. to let the extruder cool down. If I choose to work with the G4 S300 in the end code, the mail will be sent immediately. 
Is there another solution?

**Comment by Stady234** [comment]
yes, get the autoshutdown plugin... then you can set an auto delay after print to shutdown.
